qvm/sys-audio.sls

```diff
@@ -42,4 +58,10 @@ prefs:
 #    - require:
 #      - sls: qvm.sys-gui
 
-{% endif %}
\ No newline at end of file
+{% endif %}
+
+# AudioVM (AdminVM) with local 'rwx' permissions
+/etc/qubes-rpc/policy/include/admin-local-rwx:
+  file.append:
+    - text: |
+        {{ vmname }} @tag:audiovm-{{ vmname }} allow,target=dom0
```

Comment:
AudioVM should _not_ be allowed to change any VM configuration. This should be possible only from GuiVM.

Reply:
I switched to 'ro'.